Chatify/chat/api.py

```diff
@@ -49,6 +51,15 @@ def post(self, request, *args, **kwargs):
                 login(request, user)
                 user.is_online = True
                 user.save()
+                # Here using consumer we update the list
```

Comment:
create a common function from here because same thing you are doing in the consumers